.pre-commit-config.yaml

```diff
@@ -1,5 +1,19 @@
+# Git hooks for KUDO. Install with
+# pre-commit install -t pre-commit -t commit-msg
 repos:
-  - repo: git://github.com/golangci/golangci-lint
-    rev: v1.22.0
+  - repo: local
     hooks:
       - id: golangci-lint
+        name: golangci-lint
+        description: Check that source code is linted
+        entry: make lint
+        language: system
+        stages: [commit]
```

Comment:
Any specific reason why we're switching from the golangci-lint predefined hook to our `make lint` command?

Reply:
To have a single source of truth on the version of `golangci-lint` to use. `make lint` uses `hack/install-golangcilint.sh` which sets a specific version to use.